dev-packages/eslint-plugin/package.json

```diff
@@ -4,6 +4,13 @@
   "version": "1.19.0",
   "description": "Custom ESLint rules for developing Theia extensions and applications",
   "dependencies": {
-    "@theia/core": "1.19.0"
+    "@theia/core": "1.19.0",
+    "js-levenshtein": "^1.1.6"
+  },
+  "devDependencies": {
+    "copyfiles": "^2.4.1"
+  },
+  "scripts": {
+    "prepare": "copyfiles ../../packages/core/src/common/i18n/nls.metadata.json ./data -f"
```

Comment:
Why is this necessary?

Reply:
Actually, I don't think that this is necessary. I thought that we would also release the `eslint-plugin` to npm. For this case we would need to copy the json file. However, that doesn't seem to be the case, so we can rely on the exact path of the json file in the repo.